editor/gui/editor_scene_tabs.cpp

```diff
@@ -165,7 +165,7 @@ void EditorSceneTabs::_update_context_menu() {
 
 	if (tab_id >= 0) {
 		scene_tabs_context_menu->add_separator();
-		scene_tabs_context_menu->add_item(TTR("Show in FileSystem"), EditorNode::FILE_SHOW_IN_FILESYSTEM);
+		scene_tabs_context_menu->add_icon_item(get_editor_theme_icon(SNAME("ShowInFileSystem")), TTR("Show in FileSystem"), EditorNode::FILE_SHOW_IN_FILESYSTEM);
```

Comment:
This shouldn't have been changed. No other option has icon in this menu.

Reply:
I figured the option should remain consistent across menus, and a discussion should be had about having options in that menu have icons too for consistency. But I should have brought that up.

Is there something I should do on my part to revert this part? Not sure on the process after a PR is already accepted.

Reply:
You can open a new PR that reverts this particular change or someone else can do it.

For adding more icons to that menu you should open a proposal.